src/Model/ModelManager.php

```diff
@@ -633,4 +628,18 @@ protected function camelize($property)
     {
         return str_replace(' ', '', ucwords(str_replace('_', ' ', $property)));
     }
+
+    /**
+     * @param mixed $value
+     *
+     * @return string
+     */
+    private function getValueFromType($value, Type $type, string $fieldType, AbstractPlatform $platform)
```

Comment:
You can remove the `@return` annotation and use proper type hints here

```suggestion
    private function getValueFromType($value, Type $type, string $fieldType, AbstractPlatform $platform): string
```

Reply:
Other functions in the file use "@return" and not ": type" in the function declaration. I think it's better if we use the same rules in all the file.

Reply:
That's right for old parts. We can't add the return type on existing methods, because this would be a BC break.
But if you write a new method, you should use it where possible.

After this change, your PR is ready to merge :+1: